Riot/Modules/StartChat/StartChatViewController.m

```diff
@@ -306,6 +316,97 @@ - (void)refreshParticipants
     }
 }
 
+- (BOOL) participantsAlreadyContainAnEmail
```

Comment:
Don't you think possible to hide the search bar by using this method, as soon as an email has been selected?